### DISCUSSION TITLE
#4. Setting up MySQL Database (Error: Migration is not Pending)

### DISCUSSION THREAD

**Discussion by Brody-L** [socioboard/Socioboard-5.0#320] [Q&A]
I'm having trouble completing this step, particularly section C. When I run the command `npx sequelize-cli db:seed --seed 20210303111816-initialize_application_informations.cjs` inside the Sequelize-cli folder, I get an error of "Migration is not pending: 20210303111816-initialize_application_informations.cjs"

I can't tell if this step is completed or not. I've setup MySQL and the credentials for the user match that of the credentials inside the config.json file. I have not setup MongoDB and wondered if that is really a requirement or just preference. 

Any guidance you can provide would be appreciated! Please let me know of any addition information you may need from me.

**Comment by sureshbabu1995** [comment]
Hi @Brody-L 

If you see that migration is not pending that means you've all the data, please visit the Database and check the tables.

MongoDB also required to have your social account feeds and other data.

but for normal signup & login you no need to worry about it.
